.cookiecutter-replay.json

```diff
@@ -0,0 +1,46 @@
+{
+  "cookiecutter": {
+    "Introduction": "",
+    "type": "lib",
+    "name": "setuptools-betterproto",
+    "description": "A modern setuptools plugin to generate Python files from proto files using betterproto",
+    "title": "Betterproto Setuptools plugin",
+    "keywords": "setuptools,betterproto,proto,protobuf,code-generation",
+    "github_org": "frequenz-floss",
+    "license": "MIT",
+    "author_name": "Frequenz Energy-as-a-Service GmbH",
+    "author_email": "floss@frequenz.com",
+    "python_package": "setuptools_betterproto",
+    "pypi_package_name": "setuptools-betterproto",
+    "github_repo_name": "setuptools-betterproto",
+    "default_codeowners": "(like @some-org/some-team; defaults to a team based on the repo type)",
+    "_template": "gh:frequenz-floss/frequenz-repo-config-python",
+    "_repo_dir": "/home/luca/.cookiecutters/frequenz-repo-config-python/cookiecutter",
```

Comment:
this won´t work for other user's path so we should probably amend it. I guess it needs to be done in repo-config

Reply:
Thanks, will remove!

Reply:
* https://github.com/frequenz-floss/setuptools-betterproto/pull/11